smf/clickhouse_keeper/keeper_config.xml

```diff
@@ -0,0 +1,43 @@
+<clickhouse>
```

Comment:
We can punt this for later, but this does seem like the sorta file we may want to generate dynamically in the future.

Reply:
https://github.com/oxidecomputer/omicron/issues/3824